DQMOffline/L1Trigger/python/L1TriggerDqmOfflineMC_cff.py

```diff
@@ -22,11 +22,14 @@
 
 
 # do not run the emulator client in MC
-l1TriggerDqmOfflineClient.remove(l1EmulatorMonitorClient) 
+l1TriggerDqmOfflineClient.remove(l1EmulatorMonitorClient)
 
+import DQMOffline.L1Trigger.L1TriggerDqmOffline_cff as stage2_cffs_
+stage2_cffs_.Stage2l1TriggerDqmOffline.remove(stage2_cffs_.l1tStage2EmulatorMonitorClient)
```

Comment:
if you need two configurations of  'l1tStage2EmulatorMonitorClient', then here you can clone (i guess deep copy in this case) and define a second one with your changes